grpc/elliptics.fbs

```diff
@@ -0,0 +1,105 @@
+namespace fb_grpc_dnet;
+
+// TODO: use number ids for fields
+
+// Common structs
+
+table Cmd {
+    id:[uint8];
+    group_id:uint32;
+    status:int32;
+    cmd:int32;
+    backend_id:int32;
+    trace_id:uint64;
+    flags:uint64;
+    trans:uint64;
+}
+
+// Rpcs params
+
+// Write
+
+table WriteRequestHeader {
+    cmd:Cmd (required);
+
+    ioflags:uint64;
+    user_flags:uint64;
+    timestamp:uint64;
```

Comment:
```suggestion
    data_timestamp:uint64;
```

Reply:
And also a timestamp is a pair of seconds and nanoseconds

Reply:
>> pair of seconds and nanoseconds
Is it required for grpc protocol users? If we do `sec * 10E9 + nsec`, we have reserve in `int64` (signed!) for ~250 years. I suggest to throw out time as struct, at least in new protocol: time struct has bad usability.

Reply:
It is premature optimization that breaks compatibility.